qemu/tests/hv_info_check.py

```diff
@@ -0,0 +1,34 @@
+import re
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    HyperV Enlightenments info query test:
+
+    1) Boot guest.
+    2) Query supported HyperV Enlightenments by host.
+    """
+    cpu_model_flags = params["cpu_model_flags"]
+
+    vm = env.get_vm(params["main_vm"])
+    error_context.context("Query supported HyperV Enlightenments "
+                          "by host", logging.info)
+    missing = []
+    args = {"type": "full", "model": {"name": "host",
+            "props": {"hv-passthrough": True}}}
+    output = vm.monitor.cmd('query-cpu-model-expansion', args)
+    model = output.get('model')
+    model_prop = model.get('props')
+    cpu_model_flags = cpu_model_flags.replace('_', '-')
+    cpu_model_flags = re.findall(r'hv.*?,', cpu_model_flags)
```

Comment:
```suggestion
    cpu_model_flags = [i for i in .cpu_model_flags.split(',') if 'hv' in i]
```
Then you don't have the extra `,`.

Reply:
Thanks for good suggestion, modify it.